src/breeds.ts

```diff
@@ -1306,28 +1306,261 @@ export const dangerousBreeds = [
+  'Greyhound',
+  'Brussels Griffon',
+  'Hamilton Stovare',
+  'Harrier',
+  'Hovawart',
+  'Siberian Husky',
+  'Irish Soft Coated Wheaten Terrier',
+  'Jack Russell Terrier',
+  'Keeshond',
+  'Kerry Blue Terrier',
+  'Komondor',
+  'Hungarian Kuvasz',
+  'Labrador Retriever',
+  'Lakeland Terrier',
+  'Lancashire Heeler',
+  'Scottish Deerhound',
+  'Leonberger',
+  'Lhasa Apso',
+  'Irish Wolfhound',
+  'Löwchen',
+  'Alaskan Malamute',
+  'Maltese',
+  'Manchester Terrier',
+  'Mastiff',
+  'English Mastiff',
+  'Rottweiler',
+  'Staffordshire Bull Terrier',
+  'Munsterlander',
+  'Norfolk Terrier',
+  'Norwich Terrier',
+  'Otterhound',
+  'Papillon',
+  'Parson Russell Terrier',
+  'German Shepherd',
+  'Australian Shepherd',
+  'Belgian Groenendael',
+  'Belgian Laekenois',
+  'Belgian Malinois',
+  'Belgian Tervuren',
+  'Swiss White Shepherd',
+  "Catalan Sheepdog (Gos d'Atura)",
+  'Caucasian Shepherd Dog',
+  'Anatolian Shepherd',
+  'Briard',
+  'Pyrenean Shepherd',
+  'Maremma Sheepdog',
+  'Shetland Sheepdog',
+  'Australian Cattle Dog',
+  'Swedish Lapphund',
+  'Polish Lowland Sheepdog',
+  'Petit Brabançon',
+  'Pekingese',
+  'Spanish Pointer',
+  'Portuguese Pointer',
+  'Chinese Crested Dog',
+  'American Water Spaniel',
+  'Spanish Water Dog',
+  'Irish Water Spaniel',
+  'Canaan Dog',
+  'Portuguese Water Dog',
+  'Pyrenean Mountain Dog',
+  'Pharaoh Hound',
+  'Finnish Lapphund',
+  'Czechoslovakian Wolfdog',
+  'Majorcan Sheepdog',
+  'Portuguese Water Dog (Serra da Estrela)',
+  'Peruvian Inca Orchid',
+  'German Pinscher',
+  'Miniature Pinscher',
+  'Canary Island Hound',
+  'Ibizan Hound',
+  'Pointer',
+  'Pomeranian',
+  'Hungarian Puli',
+  'Ratonero Bodeguero Andaluz',
+  'Chesapeake Bay Retriever',
+  'Flat-coated Retriever',
+  'Curly-coated Retriever',
+  'Tibetan Mastiff',
+  'Rough Collie (Scottish Collie)',
+  'Bavarian Mountain Hound',
+  'Spanish Hound',
+  'Italian Hound',
+  'Saluki',
+  'Samoyed',
+  'Saint Bernard',
+  'Schipperke',
+  'Standard Schnauzer',
+  'Giant Schnauzer',
+  'Miniature Schnauzer',
+  'Sealyham Terrier',
+  'Scottish Setter',
+  'Gordon Setter',
+  'English Setter',
+  'Irish Setter',
+  'Irish Setter (Red and White)',
+  'Shar Pei',
+  'Shiba Inu',
+  'Shih Tzu',
+  'Silky Terrier',
+  'Australian Silky Terrier',
+  'Skye Terrier',
+  'Sloughi',
+  'Soft Coated Wheaten Terrier',
+  'Dutch Spaniel',
+  'Picardy Spaniel',
+  'Tibetan Spaniel',
+  'Italian Spinone',
+  'Mittelspitz',
+  'Miniature Spitz',
+  'Finnish Spitz',
+  'Japanese Spitz',
+  'Welsh Springer Spaniel',
+  'English Springer Spaniel',
+  'Tosa Inu',
+  'Sussex Spaniel',
+  'Dachshund',
+  'Newfoundland',
+  'Australian Terrier',
+  'Czech Terrier',
+  'Scottish Terrier',
+  'Welsh Terrier',
+  'Irish Terrier',
+  'Tibetan Terrier',
+  'English Toy Terrier',
+  'Swedish Vallhund',
+  'Vizsla',
+  'West Highland White Terrier',
+  'Wheaten Terrier',
+  'Whippet',
+  'Xoloitzcuintli',
+  'Yorkshire Terrier',
+];
```

Comment:
Does it make sense to add `as const` to `breedsSpain` and `dangerousBreedsSpain`, like we have for the other two exported arrays in the file? Approving the PR, anyway.

Reply:
Great catch, thanks. Updated.